JetStreamDriver.js

```diff
@@ -336,28 +339,8 @@ class Driver {
         this.benchmarks.sort((a, b) => a.plan.name.toLowerCase() < b.plan.name.toLowerCase() ? 1 : -1);
 
         let text = "";
-        for (const benchmark of this.benchmarks) {
-            const description = Object.keys(benchmark.subScores());
-            description.push("Score");
-
-            const scoreIds = benchmark.scoreIdentifiers();
-            const overallScoreId = scoreIds.pop();
-
-            if (isInBrowser) {
-                text +=
-                    `<div class="benchmark" id="benchmark-${benchmark.name}">
-                    <h3 class="benchmark-name">${benchmark.name} <a class="info" href="in-depth.html#${benchmark.name}">i</a></h3>
-                    <h4 class="score" id="${overallScoreId}">&nbsp;</h4>
-                    <h4 class="plot" id="plot-${benchmark.name}">&nbsp;</h4>
-                    <p>`;
-                for (let i = 0; i < scoreIds.length; i++) {
-                    const scoreId = scoreIds[i];
-                    const label = description[i];
-                    text += `<span class="result"><span id="${scoreId}">&nbsp;</span><label>${label}</label></span>`
-                }
-                text += `</p></div>`;
-            }
-        }
+        for (const benchmark of this.benchmarks)
+            text += benchmark.prepareToRun();
```

Comment:
Two nits:
1. `benchmark.prepareToRun` seems to be only relevant for `isInBrowser`, so can we move 342/343 to after the early return?
2. Is there a more meaningful/descriptive name instead of `benchmark.prepareToRun`? `initScoreDiv` or something?

Reply:
1. done
2. changed to `prepareBrowserUI` and moved the benchmarks sorting out into Driver initialize.